numpy/lib/type_check.py

```diff
@@ -271,6 +278,10 @@ def isreal(x):
     --------
     >>> np.isreal([1+1j, 1+0j, 4.5, 3, 2, 2j])
     array([False,  True,  True,  True,  True, False])
+    
+    Returns True for all elements in input array which are not of complex type. 
```

Comment:
You need a blank line between text and code examples here

Reply:
fixed!